plugins/nginx-vhosts/install

```diff
@@ -22,45 +22,12 @@ ssl_session_cache shared:SSL:20m;
 ssl_session_timeout 10m;
 
 ssl_ciphers EECDH+AES128:RSA+AES128:EECDH+AES256:RSA+AES256:EECDH+3DES:RSA+3DES:!MD5;
-ssl_protocols TLSv1.2 TLSv1.1 TLSv1;
-ssl_prefer_server_ciphers on;
```

Comment:
No, but [it is installed when installing dokku](https://github.com/progrium/dokku/blob/v0.3.22/plugins/nginx-vhosts/dependencies#L9) as it uses the [PPA nginx/stable](https://launchpad.net/~nginx/+archive/ubuntu/stable).